### PR TITLE
Docs: small change  about addon promote rules

### PR DESCRIPTION
**Pull request by wangyikewxgm**
Signed-off-by: 楚岳 <wangyike.wyk@alibaba-inc.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,15 +36,19 @@ Please be aware of this contribution rules when contribute addons.
 
 - A new addon added in this repo should be put in as an experimental one unless you have test for a long time in your product environment and be approved by most maintainers.
 
-- An experimental addon which has been tested for enough can promote as a verified one.
+- An experimental addon must meet these conditions can be promoted as a verified one.
```
Comment (wonderflow) [inline README.md:39]:
```suggestion
- An experimental addon must meet these conditions to be promoted as a verified one.
```

File: README.md [hunk 2]
```diff
@@ -36,15 +36,19 @@ Please be aware of this contribution rules when contribute addons.
...
+
+    - An accessible icon url and source url defined in addon's `metadata.yaml`.
     
-  - A detail introduction for this addon in `README.md` for this addon as the [documentation](https://kubevela.net/docs/reference/addons/overview).
+    - A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
+      
+    - Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
     
-  - Better provide many useful examples in [dir](examples/)
+    - Better provide more useful examples in example [dir](examples/).
```
Comment (wonderflow) [inline README.md:51]:
```suggestion
    - It's more likely to be accepted if useful examples in example [dir](examples/) provided.
```

**Review by wonderflow — COMMENTED** [review]
考虑搞成 PR Template？

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -37,3 +37,25 @@ I have:
 - [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
 - [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
 - [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.
+
+### Contribution rules
```
Comment (wonderflow) [inline .github/PULL_REQUEST_TEMPLATE.md:41]:
```suggestion
### Verified Addon Contribution
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -37,3 +37,25 @@ I have:
...
+  - This addon must have some basic but necessary information.
+
+    - An accessible icon url and source url defined in addon's `metadata.yaml`.
+    
+    - A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
+      
+    - Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
+    
+    - It's more likely to be accepted if useful examples provided in example [dir](examples/).
+-->
```
Comment (wonderflow) [inline .github/PULL_REQUEST_TEMPLATE.md:61]: - [ ]  An accessible icon url and source url defined in addon's `metadata.yaml`
- [ ] ...

File: README.md [hunk 5]
```diff
@@ -36,15 +36,19 @@ Please be aware of this contribution rules when contribute addons.
...
+
+    - An accessible icon url and source url defined in addon's `metadata.yaml`.
     
-  - A detail introduction for this addon in `README.md` for this addon as the [documentation](https://kubevela.net/docs/reference/addons/overview).
+    - A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
+      
+    - Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
     
-  - Better provide many useful examples in [dir](examples/)
+    - It's more likely to be accepted if useful examples provided in example [dir](examples/).
```
Comment (charlie0129) [inline README.md:51]:
```suggestion
    - It's more likely to be accepted if useful examples are provided in example [dir](examples/).
```

**Review by wonderflow — APPROVED** [review]

**Pull request merged**